hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/scm/net/TestNetworkTopologyImpl.java

```diff
@@ -311,57 +311,49 @@ public void testAddRemove(NodeSchema[] schemas, Node[] nodeArray) {
       cluster.add(dataNodes[i]);
     }
     // Inner nodes are created automatically
-    assertTrue(cluster.getNumOfNodes(2) > 0);
+    assertThat(cluster.getNumOfNodes(2)).isPositive();
 
-    try {
-      cluster.add(cluster.chooseRandom(null).getParent());
-      fail("Inner node can not be added manually");
-    } catch (Exception e) {
-      assertTrue(e.getMessage().startsWith(
-          "Not allowed to add an inner node"));
-    }
+    Exception e = assertThrows(Exception.class,
+        () -> cluster.add(cluster.chooseRandom(null).getParent()));
+    assertThat(e).hasMessageStartingWith("Not allowed to add an inner node");
 
-    try {
-      cluster.remove(cluster.chooseRandom(null).getParent());
-      fail("Inner node can not be removed manually");
-    } catch (Exception e) {
-      assertTrue(e.getMessage().startsWith(
-          "Not allowed to remove an inner node"));
-    }
+    Exception e2 = assertThrows(Exception.class,
+        () -> cluster.remove(cluster.chooseRandom(null).getParent()));
+    assertThat(e2).hasMessageStartingWith("Not allowed to remove an inner node");
   }
 
   @ParameterizedTest
   @MethodSource("topologies")
-  public void testGetNumOfNodesWithLevel(NodeSchema[] schemas,
+  void testGetNumOfNodesWithLevel(NodeSchema[] schemas,
       Node[] nodeArray) {
     initNetworkTopology(schemas, nodeArray);
     int maxLevel = cluster.getMaxLevel();
     try {
```

Comment:
nit: we can use assertThrow() here and other palces.
```suggestion
    IllegalArgumentException ex = assertThrows(IllegalArgumentException.class, () -> cluster.getNumOfNodes(0));
    Assertions.assertThat(ex).hasMessageStartingWith("Invalid level");
```

Reply:
Thanks, updated (also in `testGetNodesWithLevel`).